steering/elections/2021/linsun.md

```diff
@@ -0,0 +1,17 @@
+-------------------------------------------------------------
+name: Lin Sun
+ID: linsun
+info:
+  - affiliation: solo.io
+-------------------------------------------------------------
+
+### Here's a paragraph about my contributions to Istio:
+
```

Comment:
I don't think you need these headings.  (It might matter because of the new tool we're using.)

Reply:
Removed, PTAL.  Thanks @craigbox